tests/e2e/test_formatting.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import os
+
+
+def are_contents_equal(filepath1: str, filepath2: str):
+    with open(filepath1, "r", encoding="utf-8") as f1:
+        with open(filepath2, "r", encoding="utf-8") as f2:
+            return f1.read() == f2.read()
+
+
+@pytest.mark.usefixtures("init")
+def test_formatting_basic(protostar, copy_fixture):
+    os.mkdir("./format")
```

Comment:
I don't see the need for creating an intermediate directory here, especially one named `format` which looks a bit confusing to my eye.

Maybe we could just copy files to `src` and `test` dirs?

Reply:
In e2e tests we already have some default Cairo files, imo without creating a separate directory it's even more confusing.

Reply:
You can create unformatted contract fixture in `tests.data.contracts` and use it with `ProtostarFixture` like this:

```py
def test_...(protostar: ProtostarFixture):
    protostar.init_sync()
    protostar.create_files({"./src/main.cairo": UNFORMATTED_CONTRACT})
```